README.md

```diff
@@ -270,3 +261,42 @@ The following card types are allowed:
 | Visa credit | VISA-CREDIT |
 | Visa debit | VISA-DEBIT |
 | Dankort | DANKORT |
+
+
+## Diagrams
+
+### Merchants
+
+<a href='./assets/merchant-sequence-diagram.svg'>
+  <img src="./assets/merchant-sequence-diagram.svg" />
+</a>
+
+### Payments
+
+<a href='./assets/payment-sequence-diagram.svg'>
+  <img src="./assets/payment-sequence-diagram.svg" />
+</a>
+
+### Checkout
+
+<a href='./assets/checkout-sequence-diagram.svg'>
+  <img src="./assets/checkout-sequence-diagram.svg" />
+</a>
+
+### When acquirer or issurer rejects a payment
+
+<a href='./assets/acquirer-or-issuer-reject-payment-sequence-diagram.svg'>
+  <img src="./assets/acquirer-or-issuer-reject-payment-sequence-diagram.svg" />
```

Comment:
We can't use markdown syntax for images?

Reply:
Thx, now using markdown instead of inline html